installation/standard.py

```diff
@@ -48,8 +48,8 @@
 #     If you set x11libdir (that is two ones and an el) AND x11include to point
 #     to the lib and include directories, they will be used.
 #     Otherwise a search is made for common locations.
-x11libdir=''
-x11include=''
+x11libdir='/usr/X11R6/lib'
+x11include='/usr/X11R6/include'
```

Comment:
We need png for g2clib 1.4.0 and I got it to work on my Mac and Ubuntu before I check in.   I tried to install libpng in the EXTERNALS library but it is conflicting with the system one.   libpng is now a UVCDAT requirement.   What is the best approach to add it to the system?  On linux we can easy get it, but on the MAC???